src/ledger/insert_shred.zig

```diff
@@ -81,6 +82,7 @@ pub const ShredInserter = struct {
     ///
     /// This function updates the following column families:
     ///   - [`schema.dead_slots`]: mark a shred as "dead" if its meta-data indicates
+    ///     QUESTION: What does replying a shred mean?
```

Comment:
repl**a**y stage is responsible for the verification/execution of blocks from leaders (and voting/consensus).